src/pages/workspaces/workspace/notebooks/AnalysisLauncher.js

```diff
@@ -285,7 +285,7 @@ const PreviewHeader = ({
         ] : [
           h(HeaderButton, {
             onClick: () => {
-              if (runtimeStatus === 'Running') {
+              if (runtimeStatus === 'Running' && currentRuntimeTool === toolLabel) {
```

Comment:
There was another bug Alessandro reported, where if you click the 'launch' button after clicking on a RStudio analysis with a jupyter runtime, the launch button will actually launch the Jupyter runtime and load the app in tree view. This corrects the behavior so that, if I click an rstudio analysis in the analyses tab and then click launch, the create env modal opens. 

I also tested the inverse (clicking edit on a Jupyter analysis with an RStudio runtime), and that works fine.